### PR TITLE
man: Add man page for find-remotes

### PR DESCRIPTION
**Pull request by mwleeds**

### CODE REVIEW

File: Makefile-man.am [hunk 1]
```diff
@@ -26,10 +26,11 @@ ostree-admin-status.1 ostree-admin-set-origin.1 ostree-admin-switch.1	\
 ostree-admin-undeploy.1 ostree-admin-upgrade.1 ostree-admin-unlock.1	\
 ostree-admin.1 ostree-cat.1 ostree-checkout.1 ostree-checksum.1		\
 ostree-commit.1 ostree-export.1 ostree-gpg-sign.1 ostree-config.1	\
-ostree-diff.1 ostree-fsck.1 ostree-init.1 ostree-log.1 ostree-ls.1	\
-ostree-prune.1 ostree-pull-local.1 ostree-pull.1 ostree-refs.1		\
-ostree-remote.1 ostree-reset.1 ostree-rev-parse.1 ostree-show.1		\
-ostree-summary.1 ostree-static-delta.1
+ostree-diff.1 ostree-find-remotes.1 ostree-fsck.1 ostree-init.1 \
```
Comment (jlebon) [inline Makefile-man.am]: Should this one be conditional on `ENABLE_EXPERIMENTAL_API` ?

File: man/ostree-find-remotes.xml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+<refentry id="ostree">
+
+    <refentryinfo>
+        <title>ostree find-remotes</title>
+        <productname>OSTree</productname>
+
+        <authorgroup>
+            <author>
+                <contrib>Developer</contrib>
+                <firstname>Colin</firstname>
```
Comment (jlebon) [inline man/ostree-find-remotes.xml]: That should be your name, right? :)

Comment (mwleeds) [reply man/ostree-find-remotes.xml]: I guess or maybe @pwithnall. I'm unsure as to whether it should be the author of the man page or the author of the command.

File: Makefile-man.am (hunk 1 above)
Comment (mwleeds) [reply Makefile-man.am]: Yeah I suppose so

File: man/ostree-find-remotes.xml [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                </para>
+                </listitem>
+            </varlistentry>
+
+            <varlistentry>
+                <term><option>--finders</option>=FINDERS</term>
+
+                <listitem><para>
+                  Use the specified comma separated list of finders rather than
+                  the default set. Possible values: config, lan, mount (or any
```
Comment (pwithnall) [inline man/ostree-find-remotes.xml]: `config`, `lan` and `mount` should probably be [`<literal>`](http://tdg.docbook.org/tdg/4.5/literal.html) here.

File: man/ostree-find-remotes.xml [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+                  combination thereof).
+                </para></listitem>
+            </varlistentry>
+
+            <varlistentry>
+                <term><option>--pull</option></term>
+
+                <listitem><para>
+                  Pull the updates after finding them.
```
Comment (pwithnall) [inline man/ostree-find-remotes.xml]: Pulls the most recent commit it can find.

File: man/ostree-find-remotes.xml [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+    <refsect1>
+        <title>Description</title>
+
+        <para>
+            OSTree has the ability do pulls not just from configured remote
+            servers but also from peer computers on the LAN and from mounted
+            filesystems such as USB drives. Currently this requires compiling
+            OSTree with experimental API enabled, and it requires the use of
+            Collection IDs and GPG verification.
```
Comment (pwithnall) [inline man/ostree-find-remotes.xml]: Nitpick: s/Collection IDs/collection IDs/

File: man/ostree-find-remotes.xml [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+    </refsynopsisdiv>
+
+    <refsect1>
+        <title>Description</title>
+
+        <para>
+            OSTree has the ability do pulls not just from configured remote
+            servers but also from peer computers on the LAN and from mounted
+            filesystems such as USB drives. Currently this requires compiling
+            OSTree with experimental API enabled, and it requires the use of
```
Comment (pwithnall) [inline man/ostree-find-remotes.xml]: Add a `<!-- FIXME: reword this when no longer in experimental -->` or something so we don’t forget to reword it later?

File: man/ostree-find-remotes.xml (hunk 2 above)
Comment (pwithnall) [reply man/ostree-find-remotes.xml]: You wrote the man page, so you’re the author. :-)

File: man/ostree.xml [hunk 7]
```diff
@@ -290,6 +290,14 @@ Boston, MA 02111-1307, USA.
                 </para></listitem>
             </varlistentry>
 
+            <varlistentry>
+                <term><citerefentry><refentrytitle>ostree-find-remotes</refentrytitle><manvolnum>1</manvolnum></citerefentry></term>
```
Comment (pwithnall) [inline man/ostree.xml]: Technically this change should be conditional on `--enable-experimental-api`. Doing that would be a right pain, so I suggest just commenting it out with `<!-- FIXME: Enable this once find-remotes is not experimental` until later.

**Comment by mwleeds** [comment]
Feedback addressed, thanks @jlebon @pwithnall

**Comment by cgwalters** [comment]
Looks sane to me, though would prefer to queue for next release.

**Comment by cgwalters** [comment]
@rh-atomic-bot r=pwithnall 5b0284b

**Comment by pwithnall** [comment]
> 💔 Test failed - status-atomicjenkins

Huh?
```
### COMPLETED IN 613s
>>> ci/ci-release-build.sh
*** is_release_build is set to yes ***
OK: release version is 2018.1
ERROR: release commit doesn't match version
Commit message:
man: Add man page for find-remotes

Closes: #1410
Approved by: pwithnall

Build version: 2018.1
### EXITED WITH CODE 1 AFTER 0s
```

**Comment by pwithnall** [comment]
Ah, we need `is_release_build=no` in `configure.ac` after the release. @cgwalters?

**Comment by cgwalters** [comment]
Yep want to approve https://github.com/ostreedev/ostree/pull/1417 ?

**Comment by pwithnall** [comment]
Done

**Comment by pwithnall** [comment]
bot, retest this please

**Comment by cgwalters** [comment]
@rh-atomic-bot retry

**Pull request closed without merging**